test/integration/narrow/routes/monitoring.test.js

```diff
@@ -0,0 +1,205 @@
+jest.mock('../../../../app/auth')
+
+jest.mock('../../../../app/config')
+const config = require('../../../../app/config')
+
+jest.mock('../../../../app/payments')
+const { getPaymentsByFrn: mockGetPaymentsByFrn, getPaymentsByCorrelationId: mockGetPaymentsByCorrelationId } = require('../../../../app/payments')
+
+const { holdAdmin } = require('../../../../app/auth/permissions')
+const { DATA } = require('../../../mocks/values/data')
+const createServer = require('../../../../app/server')
+
+let server
+let auth
+
+describe('monitoring test', () => {
+  beforeEach(async () => {
+    config.useV1Events = true
+    config.useV2Events = true
+
+    auth = { strategy: 'session-auth', credentials: { scope: [holdAdmin] } }
+
+    mockGetPaymentsByCorrelationId.mockResolvedValue(DATA)
+    mockGetPaymentsByFrn.mockResolvedValue(DATA)
+
+    server = await createServer()
+    await server.initialize()
+  })
+
+  afterEach(async () => {
+    await server.stop()
+  })
+
+  test('GET /monitoring route returns 403 if user not in role', async () => {
```

Comment:
Not too sure what 'not in role' means here

Reply:
The app uses role based access control -> RBAC.  If the user is not in a role that allows access, they should get a 403 - unauthorised message.